cypress-custom/integration/swap.test.ts

```diff
@@ -1,5 +1,3 @@
-import './swapMod'
```

Comment:
This is a BONUS: We were running the swap tests twice. The "Mode" file is already auto-discovered by cypress